src/index.html

```diff
@@ -13,6 +13,47 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="stars stars--0">
```

Comment:
Consider using semantic tags where possible. For example, if these 'stars' sections represent different sections of content, using a <section> tag might be more appropriate than a <div>.